lib/remotable/active_record_extender.rb

```diff
@@ -119,10 +122,6 @@ def remotable_skip_validation_on_sync?
       end
 
 
-
-      attr_reader :remote_attribute_map,
-                  :local_attribute_routes
```

Comment:
⬇️ 🎨 I think the convention of using the underscore to prefix the attributes we're using to replace class-instance variables makes sense, as it denotes that these attributes are private, internal things. But these two attributes were public methods, right? Should we follow the same underscore convention? Or should we drop the underscore to indicate that these are for public consumption? Or (since it's `attr_reader`), should we just have aliases for the reader methods, a la `alias remote_attribute_map, _remote_attribute_map`? 🤔

Reply:
I think this will answer your question: https://github.com/cph/unite/pull/949/files#diff-6ecd61439b970b0e5ae49be738861446R199 😅

You're probably right, maybe we should make a non `_` method (alias) so we don't have to change the apps using Remotable. I guess I was thinking, Remotable makes its rules and the consumers follow, but maybe since this is an underscore-led method, it's a little odd, but also signifies that, yes, you can access this, but maybe ur a haxor.